src/ContextFreeTasks.NETStandard/ContextFreeTasks.NETStandard.csproj

```diff
@@ -6,7 +6,11 @@
     <Compile Include="**\*.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="NETStandard.Library" Version="1.6.1" />
+    <PackageReference Include="System.Resources.ResourceManager" Version="4.3.0">
```

Comment:
For mono, PCL, etc., we must create another project which refers the shared project anyway. This csproj is dedicated to netstandard, thus there is no reason why this csproj avoids `NETStandard.Library`.